frontend/src/metabase/databases/components/DatabaseForm/DatabaseForm.tsx

```diff
@@ -249,11 +256,9 @@ const DatabaseFormFooter = ({
 
   const hasSampleDatabase = useSetting("has-sample-database?");
 
-  const className = "database-form-footer";
-
   if (isAdvanced) {
     return (
-      <FormFooter data-testid="form-footer" className={className}>
+      <FormFooter data-testid="form-footer" px="xl">
```

Comment:
we need different padding because the footer in the modal needs padding where the footer in the setup wizard doesn't